packages/flutter_tools/lib/src/run_hot.dart

```diff
@@ -770,17 +770,22 @@ class HotRunner extends ResidentRunner {
         reassembleTimer.elapsed.inMilliseconds);
 
     reloadTimer.stop();
-    printTrace('Hot reload performed in ${getElapsedAsMilliseconds(reloadTimer.elapsed)}.');
+    final Duration reloadDuration = reloadTimer.elapsed;
+    final int reloadInMs = reloadDuration.inMilliseconds;
+
+    analyticsParameters[kEventReloadOverallTimeInMs] = '$reloadInMs';
+    flutterUsage.sendEvent('hot', 'reload', parameters: analyticsParameters);
+
+    printTrace('Hot reload performed in $reloadInMs.');
```

Comment:
this looks like a ui regression? we used to use getElapsedAsMilliseconds() which gives units and suchlike.

(i'll fix it in my timeout patch)

Reply:
Oh, thanks. I didn't realize `getElapsedAsMilliseconds()` produces formatted string.